lib/mongoose_push/application.ex

```diff
@@ -86,6 +88,39 @@ defmodule MongoosePush.Application do
     end)
   end
 
+  defp metrics do
+    [
+      # Summary is not yet supported in TelemetryMetricsPrometheus
```

Comment:
Summary is not supported, indeed, but `Distribution` is. We should use it. Also, if counter ignores measurements, then  `mongoose_push.push.time` counter has very little sense :)